includes/process-donation.php

```diff
@@ -219,7 +220,12 @@ function give_process_form_login() {
 			 *
 			 * @since 1.0
 			 */
-			do_action( 'give_ajax_donation_errors' );
+			ob_start();
```

Comment:
@raftaar1191 Can we use WordPress core function?
https://codex.wordpress.org/Function_Reference/wp_send_json_error